package.json

```diff
@@ -12,10 +12,14 @@
       "require": "./dist/cjs/index.js"
     }
   },
+  "files": [
+    "./dist/*"
+  ],
   "scripts": {
     "develop": "tsc -w --project tsconfig-cjs.json & tsc -w --project tsconfig.json",
     "build": "tsc --project tsconfig-cjs.json & tsc --project tsconfig.json",
     "lint": "eslint src/*.tsx",
+    "prepare": "npm run lint && npm run test && npm run build",
```

Comment:
Other way around? `npm run lint && npm run test && npm run build`

Reply:
That's the way it currently is. No?

Reply:
Ugh I ment:
```
npm run lint && npm run build && npm run test
```

Reply:
Erm... does it? As long as all these things happen before publish.

Reply:
I dunno. Your tests could depend on something from the build step. If it bites us later, can change it then.